lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -276,17 +276,65 @@ struct ConvertMulToAIEVecMulElemOpPattern
     unsigned laneSize = getVectorLaneSize(resultType);
 
     if ((laneSize != 32 || resultElWidth != 16) &&
-        (laneSize != 16 || resultElWidth != 32))
+        ((laneSize != 16 && laneSize != 32) || resultElWidth != 32))
       return failure();
 
-    Type accType = getVectorOpDestType(cast<VectorType>(mulOp.getType()),
-                                       /*AIEML =*/true);
+    // Deal with the case:
+    // Transfer -
+    // %1 = arith.extsi %a : vector<32xi8> to vector<32xi32>
+    // %2 = arith.extsi %b : vector<32xi8> to vector<32xi32>
+    // %3 = arith.muli %1, %2 : vector<32xi32>
+    // to -
+    // aievec.mul_elem(%a, %b) : vector<64xi8>, vector<64xi8>, vector<32xi32>
+    if (laneSize == 32 && resultElWidth == 32) {
```

Comment:
Are you planning to support this also with this conversion? Why not?
%1 = .. : vector<32xi8>
%2 = .. : vector<32xi8>
%3 = arith.muli %1, %2: vector<32xi8>

Reply:
Supporting i8 input and i8 output will be in the next change set.